environments/datasets.py

```diff
@@ -86,6 +90,23 @@ def __getitem__(self, i):
         return rolls.transpose((0, 1, 4, 2, 3))
 
 
+class EnvironmentLoader(Dataset):
+    def __init__(self, root_dir):
+        self.root_dir = root_dir
+        self.file_list = os.listdir(root_dir)
+        self.data_mean = .5
+        self.data_std = .5
+
+    def __len__(self):
+        return len(self.file_list)
+
+    def __getitem__(self, i):
+        rolls = np.load(os.path.join(
+            self.root_dir, self.file_list[i]))['arr_0']
+        rolls = (rolls - self.data_mean) / self.data_std
```

Comment:
Dont normalize the data, leave it from 0 to 1

Reply:
fixed, now mean and std are set as a parameter

Reply:
Agree with @OleguerCanal, normalizations should be done with  `transform.Normalize()`